testing/test_deploy_app.py

```diff
@@ -101,13 +130,52 @@ def insert_ssl_cert(args):
     shutil.rmtree(SSL_DIR)
   os.mkdir(SSL_DIR)
   logging.info("donwload ssl cert and insert to GKE cluster")
-  os.system("gsutil cp gs://%s/%s/* %s" % (SSL_BUCKET, args.cert_group, SSL_DIR))
-  os.system("gcloud container clusters get-credentials %s --zone %s --project %s" % (args.deployment, args.zone, args.project))
-  os.system("kubectl create -f %s" % SSL_DIR)
+  try:
+    util_run(("gsutil cp gs://%s/%s/* %s" % (SSL_BUCKET, args.mode, SSL_DIR)).split(' '))
```

Comment:
nit: Add a TODO to switch to using the GCS client instead of shelling out to gsutil.